src/App.js

```diff
@@ -1,13 +1,12 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import "./App.css";
 
+import Image from "./Components/Image.js";
+
 function App() {
   return (
     <div className="App">
-      <p>
-        Read through the instructions in the README.md file to build your NASA
-        app! Have fun <span role="img" aria-label='go!'>🚀</span>!
-      </p>
+      <Image />
```

Comment:
Since you decided to render all your API information (title, image, description, etc) in one component. Try to keep the naming of your components coherent. Think... Can this component be named better or more specific? In this case < Image />  tell's me (On first instinct) that you are rendering an image component that only contains an image. How do you think this component can be named better?